docs/installation.md

```diff
@@ -446,7 +446,8 @@ openebs-snapshot-operator-5bdcdc9b77-v7n4w     2/2     Running   0          97s
 
 The control plane pods `openebs-provisioner`, `maya-apiserver` and `openebs-snapshot-operator` should be running. If you have configured nodeSelectors , check if they are scheduled on the appropriate nodes by listing the pods through `kubectl get pods -n openebs -o wide`
 
- 
+> The resiliency of the control plane can be verified via `litmus` using <a href="https://docs.litmuschaos.io/docs/next/openebs-control-plane-validation/"> openebs-control-plane-validation </a> experiment.
```

Comment:
n_: the term validation is misleading. validation could indicate only verifying that services are configured and running. Where the validation with Litmus actually involves running chaos tests, potentially performing deletes and could result in an unstable state. 

The chaos link - should also include some help on how to recover from an unstable state or at least information on how to get support if things don't go as expected.